### PR TITLE
New post: ERF2019.

### PR DESCRIPTION
**Pull request by BorjaOuterelo**
Add new post regarding ERF presentation.
Add pdf from slides.
Will update the post once the video is available in youtube.

### CODE REVIEW

File: _posts/2019-03-20-micro-ROS_at_ERF2019.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+---
+title: micro-ROS at European Robotics Forum 2019
+author: borjaouterelo
+---
+
+The [European Robotics Forum](https://www.eu-robotics.net/robotics_forum/), the most influential meeting of the robotics community in Europe, was held in Bucharest, Romania, in March 2019. Over 900 European robotics top experts attended the 10th edition of the conference.
+
+OFERA consortium, taking advantage of the opportunity,  arrange one of their faces to face meetings the days before the ERF as some of the members will also be attending to the forum.
```
Comment (ralph-lange) [inline _posts/2019-03-20-micro-ROS_at_ERF2019.md]: arranged (past tense)

File: _posts/2019-03-20-micro-ROS_at_ERF2019.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+In the forum and as part of the TG Software & System Engineering Meeting, Ingo Luetkebohle from Bosch and Borja Outerelo from eProsima presented an overview of the micro-ROS project, exposing the challenges it faces and the involvement of the community via a demo and the ROS 2 Embedded SIG.
+
+The micro-ROS community demo was presented, making use of a Kobuki, one of the project reference platforms, Olimex STM32 E407 and micro-ROS stack. This community use case was also present in the exibitions area, as part of the Bosch booth.
+
+### Slides 
+
+<embed src="/download/OFERA-Presentation_ERF2019.pdf" type="application/pdf" width="100%" height="450px"/>
```
Comment (ralph-lange) [inline _posts/2019-03-20-micro-ROS_at_ERF2019.md]: In the slides, please change microros to micro-ros in all urls (at least three times).

File: _posts/2019-03-20-micro-ROS_at_ERF2019.md (hunk 1 above)
Comment (BorjaOuterelo) [reply _posts/2019-03-20-micro-ROS_at_ERF2019.md]: fixed

File: _posts/2019-03-20-micro-ROS_at_ERF2019.md (hunk 2 above)
Comment (BorjaOuterelo) [reply _posts/2019-03-20-micro-ROS_at_ERF2019.md]: Fixed, didn't realise, thanks.

Comment (ralph-lange) [reply _posts/2019-03-20-micro-ROS_at_ERF2019.md]: Hi Borja, for the QR code, you may copy+paste an updated version from the last slide of https://micro-ros.github.io/download/Bringing_ROS_2_on_MCUs_ASD2019.pdf

File: _posts/2019-03-20-micro-ROS_at_ERF2019.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+title: micro-ROS at European Robotics Forum 2019
+author: borjaouterelo
+---
+
+The [European Robotics Forum](https://www.eu-robotics.net/robotics_forum/), the most influential meeting of the robotics community in Europe, was held in Bucharest, Romania, in March 2019. Over 900 European robotics top experts attended the 10th edition of the conference.
+
+OFERA consortium, taking advantage of the opportunity, arranged one of their faces to face meetings the days before the ERF as some of the members will also be attending to the forum.
```
Comment (ralph-lange) [inline _posts/2019-03-20-micro-ROS_at_ERF2019.md]: Typo: "faces to face" should be "face to face". Often written with hyphens as "face-to-face".

Comment (BorjaOuterelo) [reply _posts/2019-03-20-micro-ROS_at_ERF2019.md]: I fixed both notes.

**Pull request merged**